features/step_definitions/litigator_hardship_claim_steps.rb

```diff
@@ -0,0 +1,19 @@
+Then(/^I should be on the litigator new hardship claim page$/) do
```

Comment:
**absolutely not a blocker** (given circleci slowness)- but i have started using the newer cucumber step definition where possible

Then("I should be on the litigator new hardship claim page {string}") do | param_here|

Reply:
I just copied the new AGFS hardship step from https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/0230016abe466bcac71003805c8036b5dbed9204/features/step_definitions/advocate_hardship_claim_steps.rb#L1-L3

Reply:
There's lots of these duplications, maybe a refactor when time is less pressed! 😄